VirtoCommerce.Platform.Core/Common/CancellationTokenWrapper.cs

```diff
@@ -0,0 +1,19 @@
+using System.Threading;
```

Comment:
Could you explain why we need it?

Reply:
I've added comments inline in code.

Reply:
So then if it required only for Hangfire jobs, maybe better will be moving this wrapper and abstraction to Jobs namespace?

Reply:
I'm sorry, I don't see how that could work, we need it in the indexing manager right?

Reply:
Well, I saw something similar in your PR, it may be just duplicate. If yes just need to remove it.

Reply:
This wrapper is to support System.Threading.CancellationTokens, the other wrapper is for Hangfire and is placed in the job namespace.